### DISCUSSION TITLE
Adding Media Field image path to Lucene Index

### DISCUSSION THREAD

**Discussion by mrpavlitzky** [OrchardCMS/OrchardCore#13032] [Q&A]
Hello there,

I'm working on a following use case:

I have pretty complex entities which combine many different fields and content parts. What I'd like is to have a kind of a card component on the frontend which shows only the name, short description, title and an image contained in the entity and list the cards on the page. What I want to avoid tho is querying all Documents on the page load since I don't need the 80% of the entities' content parts and fields while listing the cards.

For text fields I simply added them to the lucene index to retrieve them from there, but I struggle with finding a way of adding media path to index to retrieve it using a Razor helper. Is there a way to do that? 

To extend on that discussion - is it even reasonable to try minimize the data flowing between db and server, by loading only fields necessary for the view model while using SQLite in production?

**Comment by smemis** [comment]
i am also having the same problem. 
This is really important, there must be a way. maybe @sebastienros  will help.


I have a lucene query with that schema

{
"type": "object",
"properties": {
"Person.FirstName" : {
"type" : "string",
"description" : "Description"
},
"Person.LastName" : {
"type" : "string",
"description" : "Description"
},
"Person.Photo" : {
"type" : "string",
"description" : "Description."
}
}
}

Person.Photo is Person Content Type - Photo Content Field (Media Field).

But it's return null value because does not indexing.

This is a big problem, how can i do this.
